snuba/utils/metrics/decorators.py

```diff
@@ -0,0 +1,20 @@
+from functools import wraps
+from typing import Callable, TypeVar
+
+from snuba.utils.metrics.backends.abstract import MetricsBackend
+
+
+T = TypeVar("T")
+
+
+def track_calls(
```

Comment:
This could be a timing down the line, but that's not really necessary for this use case so avoiding the overhead.